service/profile_master_service.go

```diff
@@ -0,0 +1,52 @@
+package service
+
+import (
+	"context"
+
+	"github.com/Kotlang/authGo/db"
+	pb "github.com/Kotlang/authGo/generated"
+	"github.com/SaiNageswarS/go-api-boot/auth"
+	"github.com/SaiNageswarS/go-api-boot/logger"
+	"github.com/jinzhu/copier"
+	"go.uber.org/zap"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+type ProfileMasterService struct {
+	pb.UnimplementedProfileMasterServer
+	db *db.AuthDb
+}
+
+func NewProfileMasterService(authDB *db.AuthDb) *ProfileMasterService {
+	return &ProfileMasterService{
+		db: authDB,
+	}
+}
+
+func (s *ProfileMasterService) GetProfileMaster(ctx context.Context, req *pb.GetProfileMasterRequest) (*pb.ProfileMasterResponse, error) {
+	_, tenant := auth.GetUserIdAndTenant(ctx)
+
+	language := req.Language
+	if language == "" {
```

Comment:
Pls check standard way of checking if a string is empty in golang.. may be do trim and check if len of string is 0...

Reply:
Ok